src/data/accessibility-quiz.ts

```diff
@@ -1,4 +1,159 @@
 const accessibilityQuiz = [
+  {
+    Question: "What role does the 'alt' attribute play when adding alternative text to images on a webpage?",
+    Answer: 
+      "The 'alt' attribute describes the content of the image to users who cannot see it.",
+    Distractor1: "The 'alt' attribute enhances image aesthetics.",
+    Distractor2: "The 'alt' attribute speeds up image loading.",
+    Distractor3: "The 'alt' attribute provides translations for the image content.",
+    Explanation: 
+      "The 'alt' attribute in alternative text describes image content to users who cannot see the image.",
+    Link: "https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/"
+  },
+  {
+    Question: "What is the purpose of the 'Chrome Accessibility Developer Tools' extension for Google Chrome?",
+    Answer: 
+    "This extension adds an Accessibility sidebar pane to the Chrome Developer tools and can be used to check accessibility properties of web elements.",
+    Distractor1: "It's a new JavaScript framework",
+    Distractor2: "It's a sorting algorithm",
+    Distractor3: "It's a type of programming paradigm",
```

Comment:
let's make the wrong answers more related to accessibility instead of completely off topic.
that will make for a more engaging quiz because right now it is really obvious what the right answer is

```suggestion
    Distractor1: "It's a tool to only fix color contrast errors",
    Distractor2: "It's a tool to only fix font size errors",
    Distractor3: "It's a tool to audit only keyboard accessibility errors",
```

Reply:
ofcourse, that make more sense.